src/main/io/beeper.h

```diff
@@ -119,4 +118,7 @@ uint32_t beeperModeMaskForTableIndex(int idx);
 const char *beeperNameForTableIndex(int idx);
 int beeperTableEntryCount(void);
 bool isBeeperOn(void);
+
+#ifdef USE_DSHOT
 timeUs_t getLastDshotBeaconCommandTimeUs(void);
+#endif
```

Comment:
Macro not needed here
```suggestion
timeUs_t getLastDshotBeaconCommandTimeUs(void);
```